platform/o.n.swing.plaf/src/org/netbeans/swing/plaf/LFCustoms.java

```diff
@@ -423,4 +423,11 @@ private void disposeValues (Object[] arr) {
     public static final String PROGRESS_CANCEL_BUTTON_ICON = "nb.progress.cancel.icon";
     public static final String PROGRESS_CANCEL_BUTTON_ROLLOVER_ICON = "nb.progress.cancel.icon.mouseover";
     public static final String PROGRESS_CANCEL_BUTTON_PRESSED_ICON = "nb.progress.cancel.icon.pressed";
+
+    /**
+     * Keys used by the options dialog module.
+     */
+    protected static final String OPTIONS_USE_UI_DEFAULT_COLORS = "Nb.options.useUIDefaultsColors";
```

Comment:
Why protected instead of public?
"Nb" or "nb"? Yes I've seen both prefixes, however it would be good to stick with "nb.options"